mod.spec.js

```diff
@@ -1,10 +1,9 @@
 const assert = require('chai').assert;
-const KeyCodeLib = require('../index');
-const KeyCodeDist = require('../dist/keycode.min.js');
+
+const KeyCodeLib = require('./dist/keycode.cjs.js');
 
 // Run tests.
-test(KeyCodeLib, 'KeyCode library (Common JS / EcmaScript imported module)');
-test(KeyCodeDist, 'KeyCode generated build (Minified dist file)');
```

Comment:
.

Reply:
This won't work because we are creating an IIFE build for the browser. We need another way to test it. In the previous implementation, we had created an UMD build which supports both the Browser and CommonJS. This basically meant that when we required the minified UMD bundle, NodeJS treated it as a CommonJS module. So the tests did not make a big difference.

I have skipped creating a UMD build. The IIFE build is relatively smaller than the UMD build for the browser.

Reply:
Understood. Have created another issue for it that we'll do - https://github.com/kabirbaidhya/keycode-js/issues/25